src/app/project/project.component.html

```diff
@@ -0,0 +1,114 @@
+      <ion-icon *ngIf="level.is_locked" name="lock" color="medium" class="icon-lock"></ion-icon>
+      </div>
+    </ion-col>
+  </ion-row>
+</ion-grid>
+    
+
+<ion-content color="light">
+ <ion-list class="project-list" >
+    <div #milestones >
+      <ion-item *ngFor="let level of levels ; trackBy:trackLevel" >
+        <div color="light" class="project-card">
+          <div class="milestone" padding >
+            <ion-text color="medium">MILESTONE</ion-text>
+            <p class="project-title">{{level.name}}</p>
+            <div class="milestone-description" [innerHtml]="level.description"></div>
+            <div *ngIf="level.Activity">
+              <ion-list class="activity-list">
+                <ion-item class="activity-item" *ngFor="let activity of level.Activity">
+                  <ng-container *ngIf="!activity.is_hidden">
+                    <div *ngIf="!activity.is_locked">
+                      <!-- remove comment when milestone done -->
+                      <ion-icon *ngIf="activity.progress >= 1" name="checkmark-circle" class="icon-done" margin-end></ion-icon>
+        
+                      <circle-progress margin-end *ngIf="activity.progress < 1"
+                      [percent]="activity.progress * 100"
+                      [backgroundStrokeWidth]= "0"
+                      [radius]= "5"
+                      [space]= "-20"
+                      [maxPercent]= 100
+                      [backgroundColor]= "'#eaeaea'"
+                      [backgroundStroke]= "'#004987'"
+                      [backgroundStrokeWidth]= "0"
+                      [outerStrokeColor]= "'#2bbfd4'"
+                      [outerStrokeWidth]= "10"
+                      [showTitle]= false
+                      [showSubtitle]= false
+                      [showUnits]= false
+                      [showInnerStroke]= false
+                      [startFromZero]= false 
+                      ></circle-progress>
+                    </div>
+                    <ion-icon *ngIf="activity.is_locked" name="lock" color="medium" class="icon-lock"></ion-icon>
+                    <ion-card *ngIf="!activity.lead_image"
+                      color="primary"
+                      class="card practera-card"
+                      [ngClass]="activity.is_locked ? 'locked-card': ''"
+                      (click)="!activity.is_locked && goto(activity.id)"
+                      >
+                      <ion-card-content [ngClass]="{'feedback-card': activity.hasFeedback }">
+                        <h4 color="primary-contrast">Activity</h4>
+                        <h2 color="primary-contrast" class="activity-name">{{activity.name}}</h2>
+                        <div class="feedback-label" *ngIf="activity.hasFeedback">
+                          <p>Feedback Available</p>
+                        </div>
+                      </ion-card-content>
+                    </ion-card>
+                    <ion-card *ngIf="activity.lead_image"
```

Comment:
If the only difference between these two block of code are just the class name `image-card`, do a condition check just for that, don't repeat the whole block of code